styles/seize-bootstrap.scss

```
@@ -8,14 +8,53 @@ $orange: #ffa500;
-  $green: $green,
-  $blue: $blue,
-  $orange: $orange,
-  $yellow: $yellow
-);
+// Import Bootstrap's full SCSS stack with explicit package paths. This avoids
+// Windows/Turbopack resolving Bootstrap's bare "progress" import to the npm
+// progress JavaScript package.
+@import "bootstrap/scss/mixins/banner";
+@include bsBanner("");
+
+@import "bootstrap/scss/functions";
```

Stylelint (17.11.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)